src/Lykke.Job.ServicesMonitoring.Services/Lykke.Job.ServicesMonitoring.Services.csproj

```diff
@@ -16,8 +16,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Lykke.Common" Version="1.1.93" />
-    <PackageReference Include="RestSharp.NetCore" Version="105.2.3" />
+    <PackageReference Include="Lykke.Common" Version="3.0.2" />
```

Comment:
Need to update infrastructure nugets to at least minimal allowed in TC versions, ideally - to latest versions.